src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -20,11 +20,17 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Management.Automation;
+using System.Security;
+using Microsoft.Azure.Commands.Network.VirtualNetworkGateway;
+using Microsoft.WindowsAzure.Commands.Common;
 using MNM = Microsoft.Azure.Management.Network.Models;
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.New, "AzureRmVirtualNetworkGateway", SupportsShouldProcess = true),
+    [Cmdlet(VerbsCommon.New, 
+        "AzureRmVirtualNetworkGateway", 
+        SupportsShouldProcess = true,
+         DefaultParameterSetName = VirtualNetworkGatewayParameterSets.Empty),
```

Comment:
I would suggest a different parameter set name here (like default).  This would likely require some breaking change suppressions however, due to an issue with the breaking change tool

Reply:
Changed parameter set name to default.